src/sentry/static/sentry/app/views/projectDetail/projectDetail.tsx

```diff
@@ -131,12 +168,13 @@ class ProjectDetail extends AsyncView<Props, State> {
       selection,
     } = this.props;
     const project = this.project;
-    const hasPerformance = organization.features.includes('performance-view');
+    const {hasSessions} = this.state;
+    const hasTransactions = project?.firstTransactionEvent;
```

Comment:
```suggestion
    const hasTransactions = !!project?.firstTransactionEvent;
```

Reply:
Later in the code, we care if it's boolean or undefined.